src/clusterfuzz/_internal/tests/appengine/handlers/cron/triage_test.py

```diff
@@ -394,7 +396,56 @@ def test_no_experimental_bugs(self):
     self.mock.file_issue.return_value = 'ID', None
     for crash_type in ['Arbitrary file open', 'Command injection']:
       self.testcase.crash_type = crash_type
-      self.assertFalse(triage._file_issue(self.testcase, self.issue_tracker))
+      self.assertFalse(triage._file_issue(self.testcase, self.issue_tracker, {}, {}))
       testcase = data_handler.get_testcase_by_id(self.testcase.key.id())
       self.assertEqual('Skipping filing as this is an experimental crash type.',
                        testcase.get_metadata(triage.TRIAGE_MESSAGE_KEY))
+  
+  def test_throttle_bug(self):
+    """Test not filing issue due to throttle"""
+    bug_filed_24_hours_per_job = {}
+    bug_filed_24_hours_per_project = {}
+
+    triage._throttle_bug(self.testcase, bug_filed_24_hours_per_job, bug_filed_24_hours_per_project)
```

Comment:
May be also assert the first one can be filed?

Reply:
True, updated